sources/packages/backend/workflow/src/workflow-definitions/load-assessment-consolidated-data.bpmn

```diff
@@ -125,9 +125,11 @@
           <zeebe:header key="studentDataPartnerHasEmploymentInsuranceBenefits" value="$.data.partnerHasEmploymentInsuranceBenefits" />
           <zeebe:header key="studentDataPartnerHasFedralProvincialPDReceipt" value="$.data.partnerHasFedralProvincialPDReceipt" />
           <zeebe:header key="studentDataPartnerHasTotalIncomeAssistance" value="$.data.partnerHasTotalIncomeAssistance" />
+          <zeebe:header key="studentDataPartnerBCEAIncomeAssistanceAmount" value="$.data.partnerBceaIncomeAssistanceAmount" />
           <zeebe:header key="partner1HasEmploymentInsuranceBenefits" value="$.supportingUsers.Partner1.supportingData.hasEmploymentInsuranceBenefits" />
           <zeebe:header key="partner1HasFedralProvincialPDReceipt" value="$.supportingUsers.Partner1.supportingData.hasFedralProvincialPDReceipt" />
           <zeebe:header key="partner1HasTotalIncomeAssistance" value="$.supportingUsers.Partner1.supportingData.hasTotalIncomeAssistance" />
+          <zeebe:header key="partner1BCEAIncomeAssistanceAmount" value="$.supportingUsers.Partner1.supportingData.partnerBceaIncomeAssistanceAmount" />
```

Comment:
Please rename to `partnerBCEAIncomeAssistanceAmount`.

Reply:
The idea was to follow what was already in the form. If the other variables in the same form.io are prefixed with `partner`, it should be ok, otherwise, we should keep the form.io definition pattern.